testutils/generators_test.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/block-explorer/blob/master/LICENSE.md.
+
+package testutils
+
+import (
+	"io"
+	"testing"
+
+	"github.com/insolar/insolar/ledger/heavy/exporter"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGenerateRecords(t *testing.T) {
```

Comment:
rename the test more to be more clear. for example, `TestGenerateRecordsFunctiom()` - means testing the function GenerateRecords